components/ILIAS/Contact/classes/class.ilContactGUI.php

```diff
@@ -595,4 +566,54 @@ private function inviteToChatForm(array $options, array $usr_ids): ilPropertyFor
 
         return $form;
     }
+
+    /**
+     * @return Component[]
+     */
+    private function chatroomInvitationMessage(): array
+    {
+        $has = $this->http->wrapper()->query()->has(...);
+        if (!$has('inv_room_ref_id') || !$has('inv_usr_ids')) {
+            return [];
+        }
+
+        $inv_room_ref_id = $this->http->wrapper()->query()->retrieve(
+            'inv_room_ref_id',
+            $this->refinery->kindlyTo()->int()
+        );
+        $inv_usr_ids = $this->http->wrapper()->query()->retrieve(
+            'inv_usr_ids',
+            $this->refinery->in()->series([
+                $this->refinery->kindlyTo()->string(),
+                $this->refinery->custom()->transformation(fn(string $s): array => explode(',', $s)),
+                $this->refinery->kindlyTo()->listOf($this->refinery->kindlyTo()->int()),
+                $this->refinery->custom()->constraint(fn(array $a) => $a !== [], fn() => 'Empty array.'),
+            ])
+        );
+
+        $userlist = array_map(ilObjUser::_lookupLogin(...), $inv_usr_ids);
+
+        $url = ilLink::_getStaticLink($inv_room_ref_id, 'chtr');
+        return [
+            $this->ui_factory->messageBox()->success(
+                $this->lng->txt('chat_users_have_been_invited') . $this->ui_renderer->render(
+                    $this->ui_factory->listing()->unordered($userlist)
+                )
+            )->withButtons([
+                $this->ui_factory->button()->standard($this->lng->txt('goto_invitation_chat'), $url)
+            ])
+        ];
+    }
+
+    private function contactAction(): Closure
```

Comment:
Please add a PHPDoc describin the closure: https://phpstan.org/writing-php-code/phpdoc-types#callables